app/domain/app_status/mail.rb

```diff
@@ -16,10 +16,14 @@ class AppStatus::Mail < AppStatus
 
   def initialize
     @catch_all_inbox = catch_all_inbox
+    @truemail_working = truemail_working
   end
 
   def details
-    { catch_all_inbox: @catch_all_inbox }
+    {
+      catch_all_inbox: @catch_all_inbox,
+      truemail_working: @truemail_working
```

Comment:
we want to use this with ocp4 health checks.
shouldn't we add this to default health check instead of mail specific?